### ISSUE TITLE
dotenv module

### ISSUE THREAD

**Issue by othman-shamla** [FACG6/FENIX-BLOG#12]
there is no need to call ```dotenv``` module in every file that uses an environment variable
just require it and use it one time in ```dbConnection.js```

remove it from here
https://github.com/FACG6/FENIX-BLOG/blob/95c0acea83026d1ac5d979e351cf5a4c5ebb6f36/src/controllers/login.js#L4
https://github.com/FACG6/FENIX-BLOG/blob/95c0acea83026d1ac5d979e351cf5a4c5ebb6f36/src/controllers/midllwear/authcookies.js#L2
https://github.com/FACG6/FENIX-BLOG/blob/95c0acea83026d1ac5d979e351cf5a4c5ebb6f36/src/controllers/midllwear/protict.js#L2

**Comment by NouraldinS** [comment]
Agreed, if you are using variables from your `.env` file in any place other than `db_connection` then just call it once in your `app.js`. That'd do it.
